governance/index.rst

```diff
@@ -8,47 +8,44 @@
-three years and has resulted in serious splits within the community and
-even forks to the Bitcoin blockchain.
-
-The DGBB also provides a means for Dash to fund its own development.
-While other projects have to depend on donations or premined endowments,
-Dash uses 10% of the block reward to fund its own development. Every
-time a block is mined, 90% of the reward is split between the miner and
-a masternode per the distribution found :ref:`here <block-reward-reallocation>`,
-while the remaining 10% is not created until the end of the
-month. During the month, anybody can make a budget proposal to the
-network. If that proposal earns the net approval of at least 10% of the
-masternode network, then at the end of the month the requested amount
-will be paid out in a "superblock". At that time, the block rewards that
-were not paid out (10% of each block) will be used to fund approved
-proposals. The network thus funds itself by reserving 10% of the block
-reward for budget projects.
+Governance in a decentralized project is difficult, because by 
+definition there are no central authorities to make decisions for 
+the project. In Dash, such decisions are made by a Decentralized 
+Autonomous Organization (or DAO) consisting of the masternodes. 
+The DAO allows each masternode to vote once (yes/no/abstain) for 
+each proposal. If a proposal passes, it can then be implemented (or not) 
+by Dash's developers. A key example is early in 2016, when Dash's Core Team 
+submitted a proposal to the network asking whether the blocksize should be 
+increased to 2 MB. Within 24 hours, consensus had been reached to approve this 
+change. Compare this to Bitcoin, where debate on the blocksize has been raging 
+for nearly three years and has resulted in serious splits within the community 
+and even forks to the Bitcoin blockchain.
+
+The DAO also provides a means for Dash to fund its own development. While other 
+projects have to depend on donations or premined endowments, Dash uses 10% of 
+the block reward to fund its own development. Every time a block is mined, 90% 
+of the reward is split between the miner and a masternode per the distribution 
+found :ref:`here <block-reward-reallocation>`, while the remaining 10% is not 
+created until the end of the month. During the month, anybody can make a budget 
+proposal to the network. If that proposal earns the net approval of at least 10% 
+of the masternode network, then at the end of the month the requested amount will 
+be paid out in a "superblock". At that time, the block rewards that were not paid 
+out (10% of each block) will be used to fund approved proposals. The network thus 
+funds itself by reserving 10% of the block reward for budget projects.
+
+In late 2016, IOHK prepared a detailed report on version 0.12.1 of the Dash governance 
+system, including formal analysis of weaknesses and areas for improvement. 
+You can view the report `here <https://iohk.io/research/papers/#NSJ554WR>`__. 
```

Comment:
Just checked this and it's a broken link. Seems that IOHK pulled the doc (https://iohk.io/en/research/library/papers/dash-governance-systemanalysis-and-suggestions-for-improvement/) so I guess we need to drop the first part of this paragraph and revise the end.